.gitignore

```diff
@@ -19,8 +19,8 @@
 # This is where the result of the go build goes
 /output/**
 /output
-/_output/**
-/_output
+/_output*/**
```

Comment:
this probably only needs to be `/_output*/` (will match directories named _output/ and everything underneath), but fine as-is too.